src/identity/components/GlobalHeader/AccountDropdown.tsx

```diff
@@ -1,22 +1,30 @@
+// Libraries
 import React, {FC} from 'react'
 import {IconFont} from '@influxdata/clockface'
-import {OrganizationSummaries, UserAccount} from 'src/client/unityRoutes'
-import {
-  GlobalHeaderDropdown,
-  TypeAheadMenuItem,
-} from 'src/identity/components/GlobalHeader/GlobalHeaderDropdown'
 
+// Types
 type OrgSummaryItem = OrganizationSummaries[number]
+import {OrganizationSummaries, UserAccount} from 'src/client/unityRoutes'
 
 interface Props {
   activeOrg: OrgSummaryItem
   activeAccount: UserAccount
   accountsList: UserAccount[]
 }
 
+// Styles
 const style = {width: 'auto'}
 const menuStyle = {width: '250px'}
 
+// Components
+import {
+  GlobalHeaderDropdown,
+  TypeAheadMenuItem,
+} from 'src/identity/components/GlobalHeader/GlobalHeaderDropdown'
+
+// Constants
+import {CLOUD_URL} from 'src/shared/constants'
```

Comment:
CLOUD_URL points to /auth in cloud environments, so the URL for switching account ends up being: /auth/accounts/:accountId. 

Tested this in remocal, verified that it's the same code as is used for switch-account; and verified in prod that this is the URL that is actually hit by the switch-account page.